wouso/interface/cpanel/tests.py

```diff
@@ -295,6 +296,29 @@ def test_manage_player_view_post(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'This field is required')
 
+    def test_change_password_view_get(self):
+        p1 = self._get_player(1)
+        response = self.client.get(reverse('change_password', args=[p1.pk]))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, 'Password')
+        self.assertContains(response, 'Confirm password')
+
+    def test_change_password_view_post(self):
+        p1 = self._get_player(1)
+
+        # Check the view with a valid form
+        data = {'password': 'secret', 'confirm_password': 'secret'}
+        response = self.client.post(reverse('change_password', args=[p1.pk]), data)
+        p1 = self._get_player(1)
```

Comment:
This line should not be needed. ```p1``` is already created and the post request should change it's password according to ```data``` provided. I think that if you delete this and the test still passes you can also delete if the ```if (new == null)``` from ```_get_player()```.